packages/core/src/rules/exit_or_check.ts

```diff
@@ -7,6 +7,8 @@ import {IRuleMetadata, RuleTag} from "./_irule";
 import {ABAPFile} from "../abap/abap_file";
 
 export class ExitOrCheckConf extends BasicRuleConfig {
+  public exit: boolean = true;
```

Comment:
this is a `checkExit` setting, if true it checks for exit

as default it should report all errors, in case the configuration is just

`"exit_or_check": true`

so suggest changing these to "allowExit" and "allowCheck" instead

Reply:
alternatively if the values are undefined consider it as true